File: app/src/main/java/ffos/p3/ontologija/Ontologija.java
package ffos.p3.ontologija;

import java.io.Serializable;

public class Ontologija implements Serializable {

    public Ontologija() {
        this.sifra = sifra;
        this.naslov = naslov;
        this.tip = tip;
        this.duzina = duzina;
        this.autor = autor;
    }

    private int sifra;
    private String naslov;
    private String tip;
    private String duzina;
    private String autor;

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public String getNaslov() {
        return naslov;
    }

    public void setNaslov(String naslov) {
        this.naslov = naslov;
    }

    public String getTip() {
        return tip;
    }

    public void setTip(String tip) {
        this.tip = tip;
    }

    public String getDuzina() {
        return duzina;
    }

    public void setDuzina(String duzina) {
        this.duzina = duzina;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }
}

File: Ont-REST API/src/Peuraca/Ontologija.php
<?php

namespace Peuraca;

/**
 * @Entity @Table(name="ontologija")
 **/


class Ontologija
{
    /** @id @Column(type="integer") @GeneratedValue **/
    protected $sifra;


/**
    * @Column(type="string")
    */
    private $naslov;

    /**
    * @Column(type="string")
    */
    private $tip;

    /**
    * @Column(type="string")
    */
    private $duzina;

   
    /**
    * @Column(type="string")
    */
    private $autor;

  	public function getSifra(){
      return $this->sifra;
    }
  
    public function setSifra($sifra){
      $this->sifra = $sifra;
    }

    public function getNaslov(){
      return $this->naslov;
    }
  
    public function setNaslov($naslov){
      $this->naslov = $naslov;
    }
  
    public function getTip(){
      return $this->tip;
    }
  
    public function setTip($tip){
      $this->tip = $tip;
    }
  
    public function getDuzina(){
      return $this->duzina;
    }
  
    public function setDuzina($duzina){
      $this->duzina = $duzina;
    }
  
    public function getAutor(){
      return $this->autor;
    }
  
    public function setAutor($autor){
      $this->autor = $autor;
    }

  public function setPodaci($podaci)
	{
		foreach($podaci as $kljuc => $vrijednost){
			$this->{$kljuc} = $vrijednost;
		}
	}

}



?>
